src/auth/policy/resolver.go

```diff
@@ -1,41 +1,349 @@
 package policy
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/armon/go-radix"
 	"github.com/consensys/quorum-key-manager/src/auth/types"
 )
 
+var (
+	wildCardSuffix = "*"
+)
+
 type Operation struct {
 	Action string
 	Path   string
```

Comment:
As far as I can see from our documentation an operation would involve two elements: an action and a resource for instance `key:Get` over `stores::stores/mystore/keys/*`. if so, should we rename `Path` by `ResourcePath` for clarity?

Reply:
Updated.